sites/public/__tests__/components/listing/ListingView.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import dayjs from "dayjs"
+import { ApplicationMethodType } from "@bloom-housing/backend-core/types"
+import { render, cleanup } from "@testing-library/react"
+import { ListingView } from "../../../src/components/listing/ListingView"
+import { listing, jurisdiction } from "@bloom-housing/shared-helpers/__tests__/testHelpers"
+
+afterEach(cleanup)
```

Comment:
cleanup isn't needed anymore by the [react testing library](https://testing-library.com/docs/react-testing-library/api/#cleanup)

Reply:
done